code/workspaces/infrastructure-api/src/controllers/projectsController.js

```diff
@@ -29,9 +29,9 @@ async function getProjectByKey(request, response, next) {
 async function createProject(request, response, next) {
   const project = matchedData(request, { locations: ['body'] });
 
-  if (await projectsRepository.exists(project.projectKey)) {
+  if (await projectsRepository.exists(project.key)) {
     response.status(400).send({
```

Comment:
If this line was `return response....` then the `try` block below woudn't need to be in the `else` block. All of the responses would then need a return to avoid a linting complaint.

Reply:
You also need to return the `next()` call in the `catch` block to avoid the linting error and I didn't know whether that was the right thing to do!